src/components/NotificationModal/index.tsx

```diff
@@ -1,17 +1,11 @@
-import React from "react";
 import { useMutation } from "@apollo/client";
-import styled from "@emotion/styled";
 import Button, { Variant } from "@leafygreen-ui/button";
-import Icon from "@leafygreen-ui/icon";
-import { uiColors } from "@leafygreen-ui/palette";
-import { Body } from "@leafygreen-ui/typography";
-import { Select, Input } from "antd";
-import get from "lodash/get";
-import set from "lodash/set";
 import { Modal } from "components/Modal";
-import { RegexSelectorInput } from "components/NotificationModal/RegexSelectorInput";
-import { ErrorMessage } from "components/styles";
-import { fontSize, size } from "constants/tokens";
+import { NotificationMethod } from "components/Notifications/NotificationMethod";
```

Comment:
I almost feel like you could put the files in the `NotificationModal` and `Notifications` folders all into one folder (maybe called `Notifications`). This makes sense imo and would make these import statements less awkward

Reply:
good point. I'll keep it open to do in the next PR

Reply:
fwiw I think this would make sense to do in this PR to keep the reorganization of notifications mainly limited to one commit

Reply:
done